gtfs.yml

```diff
@@ -290,6 +290,156 @@
       text: Gondola
     - value: 7
       text: Funicular
+    - value: 100
+      text: Railway Service
```

Comment:
Why is this value duplicated?

Reply:
Good spot. And there was a word missing. Going to check the rest now.